BrowserView/lib/schema.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/**
+ *
+ * Interface to the JSON Schema processor package "ajv". It is used to check
+ * the nicknames' file.
+ *
+ * (This is a somewhat cut-back version of the general 'schemas.js' file that takes the schema from a local file and
+ * also checks the configuration file).
+ *
+ */
+
+// eslint-disable-next-line quotes
+const nicknames_schema = JSON.parse(`{
+    "title": "Schema for scribejs nickname files",
+    "description": "Nicknames for scribejs. See https://github.com/w3c/scribejs/blob/master/README.md for details",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://github.com/w3c/scribejs/blob/master/schemas/nicknames_schema.js",
+    "type": "array",
```

Comment:
At some point I'd love to make nicknames.json use a top-level object instead of an array. Using an array at the top-level inhibits things like...say...JSON-LD. 😉 It also just generally makes things harder to extend in general.

Reply:
Let us label this as a 'maybe some day...' It only makes sense if there are some serious extensions to scribe.js; for the time being it does the job. "Ain't broken, don't fix it", as they say in the US...

Reply:
But I can't make it into JSON-LD...so it's broke. 😜